### PR TITLE
Preserve GRPC service method owner in interceptors

### PR DESCRIPTION
**Pull request by sponomarev**
GRPC server [uses method objects](https://github.com/grpc/grpc/blob/aa207933cb5f62b4726bbd52bd109f9e875a8ec0/src/ruby/lib/grpc/generic/rpc_server.rb#L543-L547) and [sends them to GRPC interceptors](https://github.com/grpc/grpc/blob/aa207933cb5f62b4726bbd52bd109f9e875a8ec0/src/ruby/lib/grpc/generic/rpc_desc.rb#L56). Owner of those methods could be important and it should be the service itself. For instance, Datadog GRPC tracer [use method owner to determine a transaction name](https://github.com/DataDog/dd-trace-rb/blob/f01083af4885b8161cbee597cb313b5c0e54b595/lib/ddtrace/contrib/grpc/datadog_interceptor/server.rb#L58-L63).

Before changes they are reflected as:
image

### CODE REVIEW

File: lib/anycable/rpc_handler.rb [hunk 1]
```diff
@@ -116,5 +114,7 @@ def factory
     end
   end
 end
+
+AnyCable::RPCHandler.prepend AnyCable::Handler::CaptureExceptions
```
Comment (sponomarev) [inline lib/anycable/rpc_handler.rb]: I don't like that but can't find a better way on Friday evening.

**Comment by palkan** [comment]
After sleeping on it, I want to propose the following:
- get rid of metaprogramming
- use explicit `rescue` in service methods and `notify_exception`:

```ruby
def command(message, _unused_call)
  logger.debug("RPC Command: #{message.inspect}")
  # ...
rescue StandardError => exp
  notify_exception(exp, :command, message.to_h)
  
  CommandResponse.new(
    status: AnyCable::Status::ERROR,
    error_msg: exp.message
  )
end
```

**Comment by sponomarev** [comment]
I 'got' you. Anyway, it was a good excersice for me. 😉

**Comment by sponomarev** [comment]
@palkan bump

**Pull request merged**